docs/source/java/install.rst

```diff
@@ -83,35 +83,87 @@ Installing Nightly Packages
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <arrow.version>8.0.0.dev165</arrow.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.arrow</groupId>
+            <artifactId>arrow-vector</artifactId>
+            <version>${arrow.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.arrow</groupId>
+            <artifactId>arrow-format</artifactId>
+            <version>${arrow.version}</version>
+        </dependency>
+    </dependencies>
+
+3. Download packages needed to a temporary directory
+
+.. code-block:: shell
+
+    $ mkdir nightly-2022-03-03-0-github-java-jars
+    $ cd nightly-2022-03-03-0-github-java-jars
+    $ wget https://github.com/ursacomputing/crossbow/releases/download/nightly-2022-03-03-0-github-java-jars/arrow-vector-8.0.0.dev165.jar
+    $ wget https://github.com/ursacomputing/crossbow/releases/download/nightly-2022-03-03-0-github-java-jars/arrow-format-8.0.0.dev165.jar
+    $ tree
+    |__ arrow-format-8.0.0.dev165.jar
+    |__ arrow-vector-8.0.0.dev165.jar
+
+4. Install nightly java version to local maven repository with `mvn install:install-file`
+
+.. code-block:: shell
+
+    $ mvn install:install-file \
+        -Dfile="$(pwd)/arrow-format-8.0.0.dev165.jar" \
+        -DgroupId=org.apache.arrow \
+        -DartifactId=arrow-format \
+        -Dversion=8.0.0.dev165 \
+        -Dpackaging=jar \
+        -DcreateChecksum=true \
+        -Dgenerate.pom=true
+    [INFO] Installing /nightly-2022-03-03-0-github-java-jars/arrow-format-8.0.0.dev165.jar to /Users/arrow/.m2/repository/org/apache/arrow/arrow-format/8.0.0.dev165/arrow-format-8.0.0.dev165.jar
+    $ mvn install:install-file \
+        -Dfile="$(pwd)/arrow-vector-8.0.0.dev165.jar" \
+        -DgroupId=org.apache.arrow \
+        -DartifactId=arrow-vector \
+        -Dversion=8.0.0.dev165 \
+        -Dpackaging=jar \
+        -DcreateChecksum=true \
+        -Dgenerate.pom=true
+    [INFO] Installing /nightly-2022-03-03-0-github-java-jars/arrow-vector-8.0.0.dev165.jar to /Users/arrow/.m2/repository/org/apache/arrow/arrow-vector/8.0.0.dev165/arrow-vector-8.0.0.dev165.jar
+
+6. Validate packages installed locally on maven repository:
+
+.. code-block:: shell
+
+    $ tree /Users/arrow/.m2/repository/org/apache/arrow
+    |__ arrow-format
+        |__ 8.0.0.dev165
+            |__ arrow-format-8.0.0.dev165.jar
+            |__ arrow-format-8.0.0.dev165.pom
+    |__ arrow-vector
+        |__ 8.0.0.dev165
+            |__ arrow-vector-8.0.0.dev165.jar
+            |__ arrow-vector-8.0.0.dev165.pom
+
+5. Compile your project with `mvn clean install`.
+
+.. code-block:: shell
+
+    $ mvn clean install
+    [INFO] BUILD SUCCESS
 
 Arrow nightly builds are posted on the mailing list at `builds@arrow.apache.org`_.
```

Comment:
This should go up at the very top as part of the paragraph about GitHub. 

```
Arrow nightly builds are posted on the mailing list at `builds@arrow.apache.org`_.
The artifacts are uploaded to GitHub. For example, for 2022/03/01, they can be found at `Github Nightly`_.
```

Reply:
Added to the initial part of: Installing Nightly Packages